### DISCUSSION TITLE
Welcome to `redis-rails` Discussions!

### DISCUSSION THREAD

**Discussion by tubbo** [redis-store/redis-rails#110] [Announcements]
Hey folks, we killed the issues tab on this repo because it never represented actual issues and was usually used as a place for people to voice opinions/concerns about the redis-store gems. GitHub has something for that already, called discussions, so we're going to use them here while maintaining issue trackers for each of the dependent gems.

**Comment by alexcastrodev** [comment]
Hi @tubbo, the redis-rack-cache is maintained yet ?
